app/src/main/java/org/dosomething/letsdothis/ui/MainActivity.java

```diff
@@ -16,7 +18,6 @@
 
 public class MainActivity extends BaseActivity
```

Comment:
FIXME: There should be a toolbar when you're on the Actions, Invites and Notifications screen.